src/actions/config.js

```diff
@@ -82,6 +83,25 @@ export function configDidLoad(config) {
   };
 }
 
+const URL_BASE = 'https://raw.githubusercontent.com/netlify/netlify-cms/master/src/'
```

Comment:
You want to force everybody to rebundle each time somebody updates i18n strings? Srsly? :-1: